src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,42 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE_ARRAY = 10;
+    private int lengthArray = 0;
```

Comment:
```suggestion
    private int currentSize;
```
we don't need to set it to 0 - 0 is the default value